webpack.config.js

```diff
@@ -189,6 +189,7 @@ const config = (module.exports = {
         "/*\n* This file is subject to the terms and conditions defined in\n * file 'LICENSE.txt', which is part of this source code package.\n */\n",
     }),
     new NodePolyfillPlugin(), // for crypto, among others
+    new webpack.EnvironmentPlugin(["WEBPACK_BUNDLE"]),
```

Comment:
It turned on that we never had `Define` or `EnvironmentPlugin` plugin enabled, so `isProduction` was already `false` :(

Reply:
I might be wrong but we could accidentally remove it when migrated to webpack 5